src/content/enterprise/administration/configuration.mdx

```diff
@@ -825,6 +825,8 @@ This feature requires having a CSI driver installed in your cluster.
 - `driver`: The name of the CSI driver used when creating snapshots.
 - `class`: The VolumeSnapshotClass of the volume snapshot.
 - `storageClass`: The storage class required by volumes initialized from snapshots (optional).
+- `enableNamespaceAccessValidation`: Enable check of namespace to allow only members from the namespace to use the volume snapshot. Default false.
+- `allowIDAnnotation`: Allow use of snapshot-id as source. Default true.
```

Comment:
Is this called snapshot ID in all cloud providers?
```suggestion
- `allowIDAnnotation`: Allow using your cloud provider's snapshot ID as the source of the data. Default true.
```

Reply:
@teresaromero when this is set to false what happens? users can only use PVCs as the source?

Reply:
they can use snapshots with the namespace and name annotation, or the native k8s dataSource if the snapshot belongs to the same namespace 

```
apiVersion: v1
kind: PersistentVolumeClaim
metadata:
  annotations:
    # OPTION 1 retrieve snapshot from other namespace
    dev.okteto.com/from-snapshot-name: <name>
    dev.okteto.com/from-snapshot-namespace: <namespace>
    # OPTION 2 retrieve snapshot from id
    dev.okteto.com/from-snapshot-id: <id>
  name: pvc-from-snapshot
spec:
  storageClassName: csi-okteto-standard
  # OPTION 3 retrieve snapshot k8s native
  # snapshot has to be at the same namespace as the PVC is created
  dataSource:
    name: pvc-test-snapshot
    kind: VolumeSnapshot
    apiGroup: snapshot.storage.k8s.io
  accessModes:
    - ReadWriteOnce
  resources:
    requests:
      storage: 1Gi
```